src/native/libs/System.Security.Cryptography.Native.Apple/pal_swiftbindings.swift

```diff
@@ -193,3 +193,194 @@ public func AppleCryptoNative_IsAuthenticationFailure(error: Error) -> Bool {
     }
     return false
 }
+
+// Must remain in sync with PAL_HashAlgorithm from managed side.
+enum PAL_HashAlgorithm: Int32 {
+    case unknown = 0
+    case md5 = 1
+    case sha1 = 2
+    case sha256 = 3
+    case sha384 = 4
+    case sha512 = 5
+}
+
+enum HKDFError: Error {
+    case unknownHashAlgorithm
+}
+
+struct Logger {
```

Comment:
I hit approve-again when looking only at the diff of changing to a more classic P/Invoke style, mistakenly thinking I was looking at "changes since your last review".

The logger should, of course, be removed before checkin :smile: